Firmware/OpenLog_Artemis/menuPower.ino

```diff
@@ -58,16 +65,26 @@ void menuPower()
         settings.logA32 = false; // Disable analog logging on pin 32
       }
     }
-#if(HARDWARE_VERSION_MAJOR >= 1)
     else if (incoming == '3')
     {
-      settings.enablePwrLedDuringSleep ^= 1;
+      Serial.println(F("Please enter RTC wakeup time (sec) when put in deep sleep. 0=disable:"));
+      uint32_t tempRTCwakeUp = (uint32_t)getDouble(menuTimeout);
+      if ( tempRTCwakeUp == 0 or tempRTCwakeUp >= 60 ) // This value should be defined elsewhere.
```

Comment:
Making the minimum sleep time 60 seconds is somewhat arbitrary, and should perhaps be something like 2 seconds to match the minimum for sampling. I was also wondering if this value should be set elsewhere.